### DISCUSSION TITLE
Add Tap Action to the separator

### DISCUSSION THREAD

**Discussion by kiloptero** [Clooos/Bubble-Card#646] [Feature requests]
For any feature request you can open a new discussion here:  
https://github.com/Clooos/Bubble-Card/discussions/categories/feature-requests

For any question you can open a new discussion here:  
https://github.com/Clooos/Bubble-Card/discussions/categories/q-a

**Describe the bug**  
Every action tap action I use in the separator show more info

**To Reproduce**  

**Expected behavior**  

**Screenshots**  

**YAML**  
If applicable, add any relevant YAML code.
 - type: custom:bubble-card
    card_type: separator
    name: Luces
    icon: phu:bulb-candle
    tap_action:
      action: call-service
      service: automation.trigger
      target:
        entity_id: automation.ios_event_prueba
    sub_button:
      - entity: light.luces
        show_state: true
        icon: mdi:light-recessed
        show_icon: true
        show_background: false
        show_attribute: false

```yaml
Your code here
```
  - type: custom:bubble-card
    card_type: separator
    name: Luces
    icon: phu:bulb-candle
    tap_action:
      action: call-service
      service: automation.trigger
      target:
        entity_id: automation.ios_event_prueba

**Informations (please complete the following information):**  
 - Browser/App: [e.g. chrome, safari]
 - Bubble Card version: [e.g. 2.0.4]
 - Home Assistant version: [e.g. 2024.6.4]

**Additional context**  

Thank you! 🍻

**Comment by Clooos** [comment]
Well I was not aware that it was even possible to add a tap action directly in YAML on the separator.

It seems that Bubble Card is living its own life now 👀

**Comment by kiloptero** [comment]
hahahaha ;) coool

**Comment by tan73-73** [comment]
I try to do a pop up on separator name but its not working can you help ?
      - type: custom:bubble-card
        card_type: separator
        name: Salon
        tap_action:
          action: navigate
          navigation_path: '#salon'

**Comment by K00Kai18** [comment]
Hi,

Tried to add a tap action on separator to toggle a boolean (this boolean hides or shows some sections in my pop-up card).

But it's not working, @Clooos can you please help ?

    Type: custom:bubble-card
    card_type: separator
    name: "Rez-de-Chaussée :"
    tap_action:
      action: call-service
      service: input_boolean.toggle
      service_data:
        entity_id: input_boolean.a_c_show_schedules_rdc
    sub_button:
      - name: Plannings du Rez-de-Chaussée
        entity: input_boolean.a_c_show_schedules_rdc
        show_state: false
        show_icon: true
        show_background: false
        tap_action:
          action: toggle
    styles: >
      ${subButtonIcon[0].setAttribute("icon",
      hass.states['input_boolean.a_c_show_schedules_rdc'].state === 'on' ?
      'mdi:chevron-double-up' : 'mdi:chevron-double-down')}
